### ISSUE TITLE
Inapplicable examples on Test Rule page: "Role attribute has valid value" seem to be failed examples

### ISSUE THREAD

**Issue by joshuamaker8-create** [w3c/wai-website#1374]
### Resource URL

https://www.w3.org/WAI/standards-guidelines/act/rules/674b10/

### Description

Hi

I am learning the ACT rules in order to become an accessibility tester. On the "Role attribute has valid value" page of the Test Rules for 1.3 Adaptable, the Inapplicable Examples 2, 3, and 4 seem to be failed examples, not inapplicable examples. Is this a content error, or am I misunderstanding how the rule should be applied?

**Comment by remibetin** [comment]
@joshuamaker8-create Thank you for your interest in ACT Rules.

Comments about the ACT Rules can be shared by [opening a new issue in the act-rules.github.io repository](https://www.github.com/act-rules/act-rules.github.io/issues/new/choose/).
